start.sh.in

```diff
@@ -101,5 +101,6 @@ export TRUST_PROXY=$trustproxy
 export FORCE_SSL=$forcessl
 
 cd $bindir/../../code/
-exec 1> >(/bin/logger -s -t tc-host-secrets) 2>&1
-exec node lib/main server
+/usr/bin/nohup node lib/main server \
```

Comment:
I'd rather do a double fork, but this should be fine until we can switch to using systemd.